README.md

```diff
@@ -35,7 +37,7 @@ expect(subject).events(scheduler: scheduler, disposeBag: disposeBag)
         Recorded.next(5, "Hello"),
         Recorded.next(10, "World"),
         Recorded.completed(100)
-        ]))
+     ]))
```

Comment:
I think it's aligned to the `t` and not the `.`, right ? 
```suggestion
    ]))
```

Reply:
Fixed, sure :)